bluepysnap/morph.py

```diff
@@ -20,9 +20,10 @@
 from pathlib import Path
 
 import numpy as np
-import neurom as nm
+from morphio.mut import Morphology
```

Comment:
Should we rather return read-only morphology? If user wants, he can call `as_mutable` on it.

Reply:
don't answer, I see now. it's needed for transformation.

Reply:
I need to use a mut as a loader because I will apply transformations on it. 
I am then calling `as_immutable` on it at the end of the get: 
https://github.com/BlueBrain/snap/pull/134/files#diff-93c045373ea66b3e2b3de4f54a4610d3c1b80afbc993da10756c4186ac2aa3c8R82

Reply:
Oups I responded before seeing your comment